src/queue.ts

```diff
@@ -6,10 +6,13 @@ import { ISchemeState } from './scheme'
 import { Address, ICommonQueryOptions } from './types'
 import { BN, createGraphQlQuery, isAddress, realMathToNumber } from './utils'
 
-export interface IQueueState {
+export interface IQueueStaticState {
   dao: DAO
   id: string
   name: string
+}
+
+export interface IQueueState extends IQueueStaticState {
```

Comment:
Since Queue, Vote, Stake, and Scheme all have `state(): Observable<T>` methods now, can we have them derive from `IStateful<T>`?